db/schema.rb

```diff
@@ -13,6 +13,8 @@
 ActiveRecord::Schema.define(version: 2018_11_22_105601) do
 
   # These are extensions that must be enabled in order to support this database
+  enable_extension "fuzzystrmatch"
```

Comment:
Hi @KyleMacPherson you are including changes to schema RB that don't apply to your PR